client/dashboard/profile-wizard/index.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal depdencies
+ */
+import './style.scss';
+import { H } from '@woocommerce/components';
+
+export default class ProfileWizard extends Component {
+	componentDidMount() {
+		const body = document.querySelector( '.wp-admin' );
```

Comment:
Nice solution!  Would `document.body` be faster and less error prone here?  Or is `.wp-admin` needed to target other areas?